crates/sp-executor/src/lib.rs

```diff
@@ -207,5 +208,8 @@ sp_api::decl_runtime_apis! {
 
         /// Generates a randomness seed for extrinsics shuffling.
         fn extrinsics_shuffling_seed(header: Block::Header) -> Randomness;
+
+        /// WASM bundle for execution runtime.
+        fn execution_wasm_bundle() -> Cow<'static, [u8]>;
```

Comment:
Is there a reason to be a `Cow` here, instead of just `&'static [u8]`?

Reply:
Or it is needed for a possible case where we can return owned data from some database from disk?